### DISCUSSION TITLE
How to backup and restore?

### DISCUSSION THREAD

**Discussion by alafad** [wazuh/wazuh#16697] [Q&A]
Hello , I have installed Wazuh with as all-in-one with the following script: 
`curl -sO https://packages.wazuh.com/4.4/wazuh-install.sh && sudo bash ./wazuh-install.sh -o --all-in-one`

Now I would like test a Backup and restore , but i cant find a manual which works for me...

tested things:
- https://documentation.wazuh.com/current/user-manual/files-backup/index.html (cant find restore part)
- https://groups.google.com/g/wazuh/c/csNpkyk4c5w
- https://groups.google.com/d/topic/wazuh/qWbvFIaVCjk

I'm install the aio solution first and then I copy my entire /var/ossec folder to the new server and chown all to wazuh.

I tested this process with different ways, but the manager wouldn't start..

Can you please give me a hint or could recommend me an manual?

**Comment by mdiego92** [comment]
Hello @alafad,

For that, you would need to backup the `/var/ossec/ ` and `/var/lib/wazuh-indexer/` directories, the certificates, and passwords.

In case you have any issues with recovering the data, It is possible to reindex the data into a file following this article: https://wazuh.com/blog/recover-your-data-using-wazuh-alert-backups/

Let me remark that you may be able to understand a little better the directories and how is the data managed with the procedure of our migration guide: https://documentation.wazuh.com/current/migration-guide/wazuh-indexer.html.

Please let me know if you have any remaining questions! Looking forward to your response.

**Reply by alafad** [reply]
Hello @mdiego92,

thanks for your help,   unfortunately the visualizations and Dashboards aren't there, could you please give me a hint, how can i keep these setting, if I move the host?
